src/DynamoPackages/PackageLoader.cs

```diff
@@ -544,7 +552,10 @@ public Package ScanPackageDirectory(string directory, bool checkCertificates)
 
                 var discoveredVersion = CheckAndGetPackageVersion(discoveredPackage.VersionName, discoveredPackage.Name, discoveredPackage.RootDirectory);
 
-                var existingPackage = LocalPackages.FirstOrDefault(package => package.Name == discoveredPackage.Name);
+                var existingPackage = LocalPackages.FirstOrDefault(package => (package.Name == discoveredPackage.Name) && (package.LoadState.State != PackageLoadState.StateTypes.Unloaded));
+
+                // Do we want to notify the user when we encounter unloaded packages ?
```

Comment:
maybe just log to console for now?

Reply:
hmm - actually, do you think logging this to analytics would be useful?

Reply:
Not sure if this is the best place for analytics (seeing that this is automatically done by Dynamo...and could be many...). Maybe we can add analytics on Specific user commands (Load, Unload, etc) ?